gulpfile.js

```diff
@@ -45,7 +45,7 @@ gulp.task('test', function() {
   return gulp.src(paths.js)
     .pipe(istanbul())
     .pipe(istanbul.hookRequire())
-    .on('finish', function () {
+    .on('finish', function() {
```

Comment:
I got problem of Github's pull request
I send the pull request out, but when I commit something new in my fork, it auto goes to this PR